collect_app/src/main/AndroidManifest.xml

```diff
@@ -87,7 +78,8 @@ the specific language governing permissions and limitations under the License.
         android:label="@string/app_name"
         android:largeHeap="true"
         android:supportsRtl="true"
-        android:usesCleartextTraffic="true">
+        android:usesCleartextTraffic="true"
+        android:allowBackup="true">
```

Comment:
I was initially confused why this was added but then I remembered it's true by default. I'm guessing the intention is just to make it explicit here?

Reply:
Yes, exactly. There's an Android Studio check for it and I hadn't actually fully understood the ramifications so I think it's helpful to see it explicitly listed.